examples/create-react-app-with-styled-components/src/index.js

```diff
@@ -1,16 +1,18 @@
 import * as React from 'react';
-import ReactDOM from 'react-dom';
+import { createRoot } from 'react-dom/client';
```

Comment:
Do we know if `styled-components` is compatible with React 18? I only ever tested with Emotion.

Reply:
This is a good question. I didn't noticed any problems and warnings, but this is a small project. Based on https://github.com/styled-components/styled-components/issues/3658 and https://github.com/styled-components/styled-components/issues/3333#issuecomment-950203887 looks like it is not supported fully yet. I will revert the changes to the `styled-components` examples for now and will keep an eye for updates on their repo.